data/json/npcs/TALK_FACTION_CAMP.json

```diff
@@ -57,7 +74,7 @@
   {
     "id": "TALK_CAMP_TUTORIAL_OVERVIEW",
     "type": "talk_topic",
-    "dynamic_line": "Each faction camp starts with a bulletin board.  You can 'e'xamine the bulletin board to get a list of available missions with descriptions, and to assign an NPC to work on a mission.  The bulletin board is the primary way to access the camp.\n  If both you and a companion have two-way radios, you can assign missions to the companion remotely by talking on the radio.\n  When you ask a companion to start a camp, they'll build a bulletin board at the current location, and that will establish a faction camp at that overmap tile.  A faction camp has a central tile where the bulletin board is, and possibly one or more expansions that are in adjacent tiles.\n  All of your faction camps can be stocked with food, and your companions will eat from that food when performing camp missions, or even when they're just hungry and near the camp.\n  Currently, faction camps can be created in fields, in fire stations, and in some evac shelters.",
+    "dynamic_line": "Each faction camp starts with a bulletin board.  You can 'e'xamine the bulletin board to get a list of available missions with descriptions, and to assign an NPC to work on a mission.  The bulletin board is the primary way to access the camp.\n  If both you and a companion have two-way radios, you can assign missions to the companion remotely by talking on the radio.\n  When you ask a companion to start a camp, they'll build a bulletin board at the current location, and that will establish a faction camp at that overmap tile.  A faction camp has a central tile where the bulletin board is, and possibly one or more expansions that are in adjacent tiles.\n  All of your faction camps can be stocked with food, and your companions will eat from that food when performing camp missions, or even when they're just hungry and near the camp.\n  Currently, faction camps can be created in fields, in fire stations, and in some evac shelters.  You can start a faction camp by talking to a friendly NPC who is standing on a suitable terrain, asking about faction camps, and then telling the NPC to build a faction camp.  They'll immediately make the bulletin board.  If you don't have the option to ask the NPC to build a faction camp, you're not standing at an appropriate terrain.",
```

Comment:
in fire stations, and in some evac shelters > 
in one fire station and all evac shelters.